src/Aquifer.Data/Entities/HelpDocumentEntity.cs

```diff
@@ -7,6 +7,7 @@ public class HelpDocumentEntity
     public string Title { get; set; } = null!;
     public string Url { get; set; } = null!;
     public bool Enabled { get; set; }
+    public string? ThumbnailUrl { get; set; } = null!;
```

Comment:
The `= null!` is to tell the compiler that you promise you'll set a value. In this case, since it's nullable anyway, it's not needed.